docs/migration/v4.md

```diff
@@ -1 +1,43 @@
 # Migrating to v4.0.0
+
+## Changes to rule options
+
+- [no-bare-strings](../rule/no-bare-strings.md) now has a `ignoreDefaultAllowList` option. It can be used to exclude the default allowlist. The `allowlist` you provide (as an array or through the `allowlist` key) is now merged with the default allowlist defined in the [rule](../lib/rules/no-bare-strings.js)'s `DEFAULT_CONFIG.allowlist` property'.
```

Comment:
Since this section is so large, I think we should use a header for it `### [no-bare-strings](../rule/no-bare-strings.md)`.